planning/behavior_path_planner/config/lane_change/lane_change.param.yaml

```diff
@@ -50,13 +50,13 @@
       use_predicted_path_outside_lanelet: true
       use_all_predicted_path: false
 
-      # abort
-      enable_cancel_lane_change: true
-      enable_abort_lane_change: false
-
-      abort_delta_time: 1.0                      # [s]
-      aborting_time: 5.0                         # [s]
-      abort_max_lateral_jerk: 1000.0             # [m/s3]
+      # lane change cancel
+      lane_change_cancel:
```

Comment:
Not sure should we write `lane_change_cancel` or only `cancel`.

`lane_change.lane_change_cancel` sounds wordy.

Reply:
Thanks! I addressed it.